src/audio/crossover/crossover.c

```diff
@@ -835,6 +835,11 @@ static int crossover_reset(struct comp_dev *dev)
 
 	crossover_reset_state(cd);
 
+	cd->crossover_process = NULL;
```

Comment:
would these assignments be better in ```crossover_reset_state();```

Reply:
`crossover_reset_state()` will be called in `crossover_setup()`([code](https://github.com/thesofproject/sof/blob/main/src/audio/crossover/crossover.c#L294)) and `crossover_setup()` will be called in `crossover_copy()`([code](https://github.com/thesofproject/sof/blob/main/src/audio/crossover/crossover.c#L648)) and `crossover_prepare()`([code](https://github.com/thesofproject/sof/blob/main/src/audio/crossover/crossover.c#L781)).

In these two paths, IIUC, configs and processing functions should not be cleaned up.